scripts/ci/githubchanges.ps1

```diff
@@ -0,0 +1,85 @@
+<#
+.SYNOPSIS
+    This script generates a list of changed files in a given pull request and outputs
+    that list to a file.
+.DESCRIPTION
+    Generates a file containing a list of all modified files (added/modified) in
+    the given pull request. The output file contains a list that is newline delimited, for
+    example:
+
+    Assets/MixedRealityToolkit.SDK/AssemblyInfo.cs
+    Assets/MixedRealityToolkit.Tests/PlayModeTests/ManipulationHandlerTests.cs
+
+    The file will not contain files that have been completely deleted (for example, if
+    a change deleted Assets/MixedRealityToolkit.SDK/DeletedFile.cs), that line will not exist
+    in the output file.
+
+    Note that this script assumes that the local git repo doesn't already contain the
+    target branch (e.g. mrtk_development). This is what happens by default on Azure DevOps
+    pipeline integrations with Github pull requests.
+
+    In particular, this will checkout (via this command:
+    git fetch --force --tags --prune --progress --no-recurse-submodules origin $(System.PullRequest.TargetBranch))
+.EXAMPLE
+    .\githubchanges.ps1 -OutputFile c:\path\to\changes\file.txt -PullRequestId 1234 -RepoRoot c:\path\to\mrtk -TargetBranch mrtk_development
+#>
+param(
+    # The ID of the pull request. (e.g. 1234)
+    [string]$PullRequestId,
+
+    # The target branch that the pull request will merge into (e.g. mrtk_development)
+    [string]$TargetBranch,
+
+    # The output filename (e.g. c:\path\to\output.txt)
+    [Parameter(Mandatory=$true)]
+    [string]$OutputFile,
+
+    # The root folder of the repo (e.g. c:\repo)
+    # This primarily used to filter out files that were deleted.
+    [Parameter(Mandatory=$true)]
+    [string]$RepoRoot
+)
+
+# The pull request ID might not be present (i.e. this is an adhoc build being spun up)
+# and the target branch might not be set in which case there's nothing to validate.
+if ([string]::IsNullOrEmpty($PullRequestId) -or [string]::IsNullOrEmpty($TargetBranch))
+{
+    Write-Warning "PullRequestId and TargetBranch aren't specified, skipping."
```

Comment:
```suggestion
    Write-Warning "PullRequestId or TargetBranch aren't specified, skipping."
```
I guess we should be consistent with the logic in this warning

Reply:
Yeps, good catch